frontend/src/metabase/ui/components/utils/Portal/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { Box, Portal as MantinePortal, type PortalProps } from "@mantine/core";
+export { type PortalProps } from "@mantine/core";
+
+import ZIndex from "metabase/css/core/z-index.module.css";
+
+export const Portal = ({ children, ...props }: PortalProps) => {
+  return (
+    <MantinePortal {...props}>
+      <Box pos="absolute" className={ZIndex.Overlay}>
```

Comment:
Should we allowing passing the className to this box?

Reply:
Would that be useful?

Reply:
My gut says it's not worth it - I'm not picturing the specific use case at the moment - but definitely please press me on this if you think it would be worthwhile.